ClinicWeb/src/main/java/pl/infoshare/clinicweb/patient/PatientController.java

```diff
@@ -87,9 +87,26 @@ public String editPatient(@ModelAttribute("patient") Patient patient, Model mode
     }
 
     @GetMapping("/update-patient")
-    public String fullDetailPatient(@RequestParam(value = "pesel", required = false)@ModelAttribute String pesel, Model model) {
+    public String fullDetailPatient(@RequestParam(value = "pesel", required = false) @ModelAttribute String pesel, Model model) {
         model.addAttribute("updatePatient", patientService.findByPesel(pesel));
         return "update-patient";
     }
 
+    @PostMapping("/search-patient")
+    public String searchPatientByPesel(@RequestParam("pesel") String pesel, Model model, Address address) {
+        Patient byPesel = patientService.findByPesel(pesel);
+        if (patientService.findByPesel(pesel) != null) {
+            model.addAttribute("searchForPesel", byPesel);
+        } else {
+            model.addAttribute("error", "Nie znaleziono pacjenta o podanym numerze pesel: " + pesel);
+        }
+        return "searchPatient";
+    }
+
+    @GetMapping("/searchPatient")
```

Comment:
```suggestion
    @GetMapping("/search-patient")
```
jak pokazuje jedną zmianę to ona się tyczy zawsze globalnie, całego projektu